code/modules/jobs/job_types/enclave.dm

```diff
@@ -406,7 +410,13 @@
 		/obj/item/reagent_containers/hypospray/medipen/stimpak = 2,
 		/obj/item/grenade/chem_grenade/cleaner = 1,
 		/obj/item/pda = 1,
+		)
+
+/datum/outfit/loadout/scientist_first
+	name = "Scientist Equipment"
+	backpack_contents = list(
 		/obj/item/gun/energy/gammagun = 1,
+		/obj/item/gun/energy/kinetic_accelerator/crossbow = 1,
```

Comment:
Only gripe is this. Far too powerful for an already absurdly powerful faction. People don't like the gamma gun because they're not familiar with it. Given that you can spawn gorillas in town, however, with it, that's not a big deal.